internal/jennies/golang/converter.go

```diff
@@ -35,7 +33,7 @@ func (jenny *Converter) Generate(context languages.Context) (codejen.Files, erro
 
 		filename := filepath.Join(
 			formatPackageName(builder.Package),
-			fmt.Sprintf("%s_converter_gen.go", strings.ToLower(builder.Name)),
+			fmt.Sprintf("%s_converter_gen.go", formatFileName(builder.Name)),
```

Comment:
Nit (personal, feel free to skip):
```suggestion
			formatFileName(builder.Name) + "_converter_gen.go"
```